drivers/overlay/overlay.go

```diff
@@ -542,9 +541,29 @@ func SupportsNativeOverlay(graphroot, rundir string) (bool, error) {
 	home := filepath.Join(graphroot, "overlay")
 	runhome := filepath.Join(rundir, "overlay")
 
-	if _, err := os.Stat(getMountProgramFlagFile(home)); err == nil {
+	var contents string
+	flagContent, err := ioutil.ReadFile(getMountProgramFlagFile(home))
+	if err == nil {
+		contents = strings.TrimSpace(string(flagContent))
+	}
+	switch contents {
+	case "true":
 		logrus.Debugf("overlay storage already configured with a mount-program")
 		return false, nil
+	case "":
```

Comment:
Should this be the default case.  In cases with the file does not have "true" or "false" then scan.

Reply:
Replacing `case ""` with `default`, which is what I think you're asking here.